.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,33 +12,43 @@ The following links the related issue to the PR (https://docs.github.com/en/free
 
 Fixes #\<issue_number>
 
-### Does your PR introduce any breaking changes? If yes, please list them.
+<!-- Does your PR introduce any breaking changes? If yes, please list them. -->
 
-<!-- FILL IN or None -->
+<details>
+  <summary><b>Before submitting</b></summary>
 
-## Before submitting
-
-- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
+- Was this **discussed/agreed** via a GitHub issue? (not for typos and docs)
 - [ ] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
 - [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
```

Comment:
Should these be updated too?
```suggestion
- Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- Did you make sure your **PR does only one thing**, instead of bundling different changes together?
```

Reply:
These shall be checked... :otter:

Reply:
Ah I see. I thought all of these would no longer have checkbox :)